forwards_paper.tex

```diff
@@ -544,10 +584,17 @@ \subsection*{Overview of the API}
 encoding, we can achieve data transfer rates that would be impossible under
 any text-based approach while retaining excellent portability.
 
-%%%%%%%%%%%%%%%%%%%%%%
-\section*{Results}
+% Outline:
+% 1. estimates and analytical results
+% 2. in practice: measurements from simulations
+\subsection*{Estimates of run-time complexity}
+
+For simple WF alg with exactly one recomb per gen, should be 2Nt edges; post-simplification should be 2N log N * rho edges
```

Comment:
I'm being stupid here, but where does rho come from? We have one crossover per generation, so this seems like this has been fixed already? Or is it just a function of the genome length L?

Reply:
I haven't edited this bit yet.  You're right.  Although this section might be more useful if we keep the genome-length scaling in rather than setting L=1.